source/profiles.c

```diff
@@ -61,10 +61,10 @@ static uint8_t flowValue[NUM_COLUMN] = {0,  11, 22, 33,  44,  55,  66,
 void animatedRainbowFlow(led_t *currentKeyLedColors, uint8_t intensity) {
   for (int i = 0; i < NUM_COLUMN; i++) {
     setColumnColorHSV(currentKeyLedColors, i, flowValue[i], 255, 125, intensity);
-    if (flowValue[i] == 179) {
+    if (flowValue[i] >= 179 && flowValue[i] < 240) {
       flowValue[i] = 240;
     }
-    flowValue[i]++;
+    flowValue[i] += 3;
```

Comment:
I replaced all 1 increments with 3. Previous value of 1 was too slow to test actual speed functionality.
Now with increment 1, we just call the function less, and the effect is similar - the effect is slow even with this bigger increment.

Reply:
SG